src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Builder/ScaffoldBuilder.cs

```diff
@@ -74,4 +74,7 @@ public IScaffolder Build(IServiceProvider serviceProvider)
 
         return new Scaffolder(Name, DisplayName, Category, Description, _options, steps, _stepPreparers);
     }
+
+    private static string FixName(string name)
```

Comment:
Similar to the option naming, we'll want to flesh this out and add tests around it.

Reply:
sounds good